build-manifest.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+      "name": "hopsworks/mysqld_exporter",
+      "version": "env:VERSION",
+      "dockerFile": ""
```

Comment:
I think you can remove the line with the empty `dockerFile`

Reply:
done